spec/std/crystal/compiler_rt/divmod128_spec.cr

```diff
@@ -0,0 +1,89 @@
+require "spec"
+
+# TODO: Remove these helpers in PR part 2
+
+private def make_ti(a : Int128, b : Int128)
+  (a << 64) + b
+end
+
+private def make_tu(a : UInt128, b : UInt128)
+  (a << 64) + b
+end
+
+# Specs ported from compiler-rt
+
+private def test__divti3(a : Int128, b : Int128, expected : Int128, file = __FILE__, line = __LINE__)
+  it "passes compiler-rt builtins unit tests" do
```

Comment:
I know this is the same design as in the existing `mulodi4_spec.cr`, so don't feel obliged to change this. But I don't think there's much reason for having many individual examples (with non-descriptive names). I'd propose to place all related expecations in a single example.

Reply:
Placing it all in the one example will show just the first fail instead of all.

Reply:
Sure, but that doesn't really matter much. They're all expected to pass anyways.

Reply:
Of course they are :D I'm saying is that you want to know exactly which ones are failing if they don't.